src/moneypenny/schemas/post.json

```diff
@@ -7,46 +7,99 @@
   "default": {},
   "examples": [
     {
-      "username": "jb007",
-      "uidNumber": 1007,
-      "isMemberOf": [
-	{
-	  "name": "doubleos",
-	  "id": 500
-	},
-	{
-	  "name": "staff",
-	  "id": 200
-	}
-      ]
+      "token": {
+        "data": {
```

Comment:
This is all set up to match the legacy token format, which in turn matches the NCSA-adulterated CILogon tokens. All of this is going away in the long run, so I'd rather not incorporate this mess into another application. Instead, could you define the parameters using the same names as are used in the new API documented in SQR-049? That will keep everything consistent in the future and will make it much simpler and also avoid some of the weirdness of the legacy token schema, at the cost of currently needing to convert `/auth/analyze` results to the new format (which will go away once the new Gafaelfawr is deployed).

The schema would therefore look something more like this (probably without the name since you don't care):
```
{
  "username": "alice",
  "name": "Alice Example",
  "uid": 24187,
  "groups": [
    {
      "id": 4173,
      "name": "example-group"
    },
    {
      "id": 5671,
      "name": "other-group"
    }
  ]
}
```
(example from the `/auth/api/v1/user-info` route).

Reply:
Done